en/mkdocs.yml

```diff
@@ -28,7 +28,7 @@ edit_uri: https://github.com/wso2/docs-apk/edit/main/en/docs/
 dev_addr: localhost:8000
 
 # Copyright
-copyright: WSO2 API Platform for Kubernetes - Documentation
+copyright: Copyright © WSO2 LLC (2023)Content licensed under CC By 4.0. | Sample code licensed under Apache 2.0.
```

Comment:
Need to add links. For an example see [1].
WSO2 LLC - https://wso2.com/
CC by 4.0 - https://creativecommons.org/licenses/by/4.0
Apache 2.0 - http://www.apache.org/licenses/


[1] https://apim.docs.wso2.com/en/latest/